checker/src/org/checkerframework/checker/nullness/NullnessAnnotatedTypeFactory.java

```diff
@@ -154,6 +154,13 @@ public NullnessAnnotatedTypeFactory(BaseTypeChecker checker, boolean useFbc) {
                 org.checkerframework.checker.nullness.compatqual.MonotonicNonNullType.class,
                 MONOTONIC_NONNULL);
 
+        // Alias deprecated annotation
```

Comment:
But here I can't copy the annotation values, because I don't have a concrete instance yet.
Are you suggesting enhancing the handling of declaration annotation aliases to copy over all values? Do we do that for type annotations?